test/channel/base_test.rb

```diff
@@ -92,6 +111,23 @@ def rm_rf
     assert_nil @channel.last_action
   end
 
+  test "try calling a public method defined on Base" do
+    @channel.perform_action 'action' => :kick
+    assert_nil @channel.last_action
```

Comment:
Way easier to understand those tests now, @cristianbica :+1:

Hm, I'm not sure. I can certainly see it would be nice in development if you happen to send a forbidden method. I don't see the use in production (those forbidden calls should already be long gone).

How would a forbidden response look like from the clients perspective?